.github/workflows/cd.yml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        description: 'Run the build with tmate debugging enabled'
+        required: false
+        default: false
+
+jobs:
+  build:
+    name: Build, push, and deploy
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+    #if: github.ref == 'refs/heads/main'
```

Comment:
Don't think we need this right?

```suggestion
```

Reply:
I left it as example if you want to customize it later.

Reply:
E.g. to have a nightly Docker image that's updated when pushing to main.